crates/sui/tests/snapshots/shell_tests__with_network__summaries__summarize_json.sh.snap

```diff
@@ -0,0 +1,167 @@
+object.json
+object_bag.json
+object_table.json
+package.json
+pay.json
+poseidon.json
+priority_queue.json
+prover.json
+random.json
+sui.json
+table.json
+table_vec.json
+token.json
+transfer.json
+transfer_policy.json
+tx_context.json
+types.json
+url.json
+vdf.json
+vec_map.json
+vec_set.json
+versioned.json
+zklogin_verified_id.json
+zklogin_verified_issuer.json
+address.json
+ascii.json
+bcs.json
+bit_vector.json
+bool.json
+debug.json
+fixed_point32.json
+hash.json
+macros.json
+option.json
+string.json
+type_name.json
+u128.json
+u16.json
+u256.json
+u32.json
+u64.json
+u8.json
+uq32_32.json
+uq64_64.json
+vector.json
+{
+  "root_package_id": "0x0000000000000000000000000000000000000000000000000000000000000002",
+  "root_package_original_id": "0x0000000000000000000000000000000000000000000000000000000000000002",
+  "root_package_version": 1,
+  "dependencies": {
+    "0x0000000000000000000000000000000000000000000000000000000000000001": "0x0000000000000000000000000000000000000000000000000000000000000001"
+  },
+  "linkage": {
+    "0x0000000000000000000000000000000000000000000000000000000000000001": {
+      "upgraded_id": "0x0000000000000000000000000000000000000000000000000000000000000001",
+      "upgraded_version": 1
+    }
+  },
+  "type_origins": {
+    "0x0000000000000000000000000000000000000000000000000000000000000001": [
+      {
+        "module_name": "fixed_point32",
+        "datatype_name": "FixedPoint32",
+        "package": "0x0000000000000000000000000000000000000000000000000000000000000001"
+      },
+      {
+        "module_name": "uq32_32",
+        "datatype_name": "UQ32_32",
+        "package": "0x0000000000000000000000000000000000000000000000000000000000000001"
+      },
+      {
+        "module_name": "uq64_64",
+        "datatype_name": "UQ64_64",
+        "package": "0x0000000000000000000000000000000000000000000000000000000000000001"
+      },
+      {
+        "module_name": "option",
+        "datatype_name": "Option",
+        "package": "0x0000000000000000000000000000000000000000000000000000000000000001"
+      },
+      {
+        "module_name": "ascii",
+        "datatype_name": "String",
+        "package": "0x0000000000000000000000000000000000000000000000000000000000000001"
+      },
+      {
+        "module_name": "ascii",
+        "datatype_name": "Char",
+        "package": "0x0000000000000000000000000000000000000000000000000000000000000001"
+      },
+      {
+        "module_name": "bit_vector",
+        "datatype_name": "BitVector",
+        "package": "0x0000000000000000000000000000000000000000000000000000000000000001"
+      },
+      {
+        "module_name": "string",
+        "datatype_name": "String",
+        "package": "0x0000000000000000000000000000000000000000000000000000000000000001"
+      },
+      {
+        "module_name": "type_name",
+        "datatype_name": "TypeName",
+        "package": "0x0000000000000000000000000000000000000000000000000000000000000001"
+      }
+    ]
+  }
+}{
```

Comment:
Is this kind of appending valid in all formats? Should we instead add this "extra" data into the summary itself?

Reply:
This is just an oddity of the `cat` not inserting a newline in the test (I'll see what I can do about that). These are not appended together and they reside in different output files.